core/modules/contact/contact.admin.inc

```diff
@@ -24,33 +24,40 @@ function contact_settings_form($form, &$form_state) {
   );
   $form['fields']['copy'] = array(
     '#type' => 'checkbox',
-    '#title' => t("Show the 'Send yourself a copy' checkbox"),
-    '#description' => t("Allow users to send themselves a copy on site-wide and personal contact forms."),
+    '#title' => t("Allow users to send themselves a copy of the submitted data."),
+    '#description' => t("Exposes a 'Send yourself a copy' checkbox on site-wide and personal contact forms."),
```

Comment:
Include a 'Send yourself a copy' checkbox on site-wide and personal contact forms